### PR TITLE
hide character count for #61

### PR DESCRIPTION
**Pull request by ghajba**
I think this is not the prettiest solution but I am not familiar with CS but as I have seen locally this does the trick. Perhaps I will take a deeper look at CS to improve this solution if there is a way.

### CODE REVIEW

File: lib/main.coffee [hunk 1]
```diff
@@ -54,6 +54,14 @@ module.exports =
       items:
         type: 'boolean'
       order: 7
+    hidechars:
+      title: 'Hide character count'
+      description: 'Hides the character count from the view'
+      type: 'boolean'
+      default: 'false'
```
Comment (nesQuick) [inline lib/main.coffee]: That's not a valid datatype for a `bool` value. Just remove the quotes :)

File: lib/main.coffee [hunk 2]
```diff
@@ -54,6 +54,14 @@ module.exports =
...
+      items:
+        type: 'boolean'
```
Comment (nesQuick) [inline lib/main.coffee]: Not sure what that `items` key is for. Maybe it's not needed for a single option key. Could you clarify this?

File: lib/wordcount-view.coffee [hunk 3]
```diff
@@ -15,7 +15,10 @@ class WordcountView extends View
   update_count: (editor) ->
     text = @getCurrentText editor
     [wordCount, charCount] = @count text
-    @divWords.innerHTML = "#{wordCount || 0} W | #{charCount || 0} C"
+    if atom.config.get('wordcount.hidechars')
+      @divWords.innerHTML = "#{wordCount || 0} W"
+    else
+      @divWords.innerHTML = "#{wordCount || 0} W | #{charCount || 0} C"
```
Comment (nesQuick) [inline lib/wordcount-view.coffee]: I think it's ok to handle it this way. Another possible solution would be to split the code that word and char count are appended separately, and then you could do it in a one line change. like:

```coffee
xyz.append(" | #{charCount || 0} C") unless atom.config.get('wordcount.hidechars')
```

**Comment by nesQuick** [comment]
Hey @ghajba,

thanks for your contribution. Very much appreciated. I will review it asap and leave feedback.

File: lib/main.coffee (hunk 2 above)
Comment (ghajba) [inline lib/main.coffee]: It's just a monkey see monkey do solution I copied from the previous boolean checkbox. It seems not to matter for atom in the end, it works with and without this `items`.

File: lib/main.coffee (hunk 1 above)
Comment (ghajba) [inline lib/main.coffee]: :) it seems working even with quotes ;)

**Comment by nesQuick** [comment]
Thank you very much for your contribution @ghajba  :)

**Comment by ghajba** [comment]
You are welcome! Perhaps I will do other PRs too :)

**Comment by nesQuick** [comment]
I just released 2.7.0 including your change. More PRs welcome :)

**Pull request merged**